### PR TITLE
Update config_loader.py

### PR DESCRIPTION
**Pull request by mrT23**
### **User description**
aaa

___

### **PR Type**
enhancement

___

### **Description**
- Added a print statement to log an error when settings retrieval fails in `get_settings`.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>config_loader.py</strong><dd><code>Enhance error handling in config loading</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
pr_agent/config_loader.py

- Added error logging to the `get_settings` function.

</details>
    

  </td>
  <td><a href="https://github.com/Codium-ai/pr-agent/pull/888/files#diff-35a8f47e651f179d8a5ad887b5d484e99650454fcc30efaa58945643bfc0b000">+1/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: pr_agent/config_loader.py
Comment (barnett-yuxiang) [inline pr_agent/config_loader.py:1]: nice aaa

**Review by barnett-yuxiang — APPROVED** [review]
aaa

**Comment by lingxiao001** [comment]
Preparing review...

**Comment by lingxiao001** [comment]
## PR Reviewer Guide 🔍

Here are some key observations to aid the review process:

<table>
<tr><td>⏱️&nbsp;<strong>Estimated effort to review</strong>: 1 🔵⚪⚪⚪⚪</td></tr>
<tr><td>🧪&nbsp;<strong>No relevant tests</strong></td></tr>
<tr><td>🔒&nbsp;<strong>No security concerns identified</strong></td></tr>
<tr><td>⚡&nbsp;<strong>Recommended focus areas for review</strong><br><br>

<details><summary><a href='https://github.com/Codium-ai/pr-agent/pull/888/files#diff-35a8f47e651f179d8a5ad887b5d484e99650454fcc30efaa58945643bfc0b000R39-R39'><strong>Inadequate Error Logging</strong></a>

Using `print("error")` for error handling is not ideal. The message is generic and provides no context about what failed. It's recommended to use a proper logging library (like Python's `logging` module) and log the actual exception to provide more details for debugging.
</summary>

```python
print("error")
```

</details>

</td></tr>
</table>

**Comment by lingxiao001** [comment]
**[Persistent review](https://github.com/qodo-ai/pr-agent/pull/888#issuecomment-3536184462)** updated to latest commit https://github.com/qodo-ai/pr-agent/commit/254c3e2ed14bd4fc6a38bee50e0efc5bf115cb88

**Comment by lingxiao001** [comment]
Preparing review...

**Comment by lingxiao001** [comment]
**[Persistent review](https://github.com/qodo-ai/pr-agent/pull/888#issuecomment-3536184462)** updated to latest commit https://github.com/qodo-ai/pr-agent/commit/254c3e2ed14bd4fc6a38bee50e0efc5bf115cb88

**Pull request closed without merging**